aspnetcore/security/authorization/secure-data/samples/final3/Pages/Contacts/Details2.cshtml

```diff
@@ -0,0 +1,61 @@
+@page "{id:int}"
```

Comment:
This is just a copy of `Details`, with editing functionality removed. We're just demoing the `OnGet` handler so there's no need for the `OnPost` and calls to it.